decidim-assemblies/app/controllers/decidim/assemblies/assemblies_controller.rb

```diff
@@ -2,19 +2,13 @@
 
 module Decidim
   module Assemblies
-    # A controller that holds the logic to show Assemblies in a
-    # public layout.
+    # A controller that holds the logic to show Assemblies in a public layout.
     class AssembliesController < Decidim::Assemblies::ApplicationController
       include ParticipatorySpaceContext
       participatory_space_layout only: :show
+      include FilterResource
 
-      helper Decidim::AttachmentsHelper
-      helper Decidim::IconHelper
-      helper Decidim::WidgetUrlsHelper
```

Comment:
I moved all the helpers inside each respective base helper and leaved only the concerns in the controller.
I don't know if it's good practice or not. It seemed like a good idea at the moment (less clutter), but can be changed if the reviewers ask for it.